perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh

```diff
@@ -1,29 +1,36 @@
 #!/bin/bash
 set -e
 sudo apt-get update
+
+echo Installing git
+sudo apt-get install git
+echo Installing go-lang 1.19.5
+wget -O go_tar.tar.gz https://go.dev/dl/go1.19.5.linux-amd64.tar.gz
+sudo rm -rf /usr/local/go && tar -xzf go_tar.tar.gz && sudo mv go /usr/local
+export PATH=$PATH:/usr/local/go/bin
 echo Installing fio
 sudo apt-get install fio -y
-echo Installing gcsfuse
-GCSFUSE_VERSION=0.41.12
-curl -L -O https://github.com/GoogleCloudPlatform/gcsfuse/releases/download/v$GCSFUSE_VERSION/gcsfuse_"$GCSFUSE_VERSION"_amd64.deb
-sudo dpkg --install gcsfuse_"$GCSFUSE_VERSION"_amd64.deb
-cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse/perfmetrics/scripts"
+
+cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse"
+# Get the latest commitId of yesterday in the log file
+git log --before='yesterday 23:59:59' --max-count=1 --pretty=%H >> log.txt
```

Comment:
instead of writing the output to file and then doing a cat on that file, check if you can get the output directly to variable

Reply:
fixed